components/feature/Button/FloatingBtn/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { BottonIcon, TopIcon } from 'public/static/icons';
+import { FloatingBtnContainer } from './styled';
+
+interface Props {
+	isTop: boolean; // true->to top, false->to bottom
+}
+const FloatingBtn = (props: Props) => {
+	const handleScroll = (isTop: boolean) => {
+		if (isTop) {
+			// 탑 버튼 로직
+			window.scrollTo({
+				top: 0,
+				behavior: 'smooth',
+			});
+		} else {
+			// 바텀 버튼 로직
+			window.scrollTo({
+				top: document.documentElement.scrollHeight,
+				behavior: 'smooth',
+			});
+		}
+	};
```

Comment:
```suggestion
	const handleScroll = (moveTo: 'top' | 'down') => {
		window.scrollTo({
				top: moveTo === 'top' ? 0 : document.documentElement.scrollHeight,
				behavior: 'smooth',
			});
	};
```

위와 같은 방식으로하면 굳이 분기처리를 사용하지 않아도 될 것 같네요 ~

추가적으로, document 객체에 직접 접근하는 것을 최대한 "지양" 하는 것이 좋다고 생각하는 데, 리뷰하는 시점에서 대안책이 생각이 안나지만, 방법이 있다면 대체해보는 것도 좋을 것 같아요 ~

Reply:
> 

사실  이거 방법 찾아보느라 리뷰 반영이 좀 늦었는데, 현재까지는 `Ref 를 사용하는 방안`이 최선,, 근데 일단 Footer 컴포넌트에 Ref 적용해서 Ref 적용된 컴포넌트로 이동한다 라는 로직.. 을 사용하려면 입점신청 페이지 컴포넌트에서 함수를 만들어서 탑바텀으로 넘겨주는 식으로 해야할 것 같아서 다른 방법 있는지 찾아보고 추후 수정해보겠습니다.. 일단 추천해준 대로 함수 바꿔놓으께 ~

Reply:
> 위와 같은 방식으로하면 굳이 분기처리를 사용하지 않아도 될 것 같네요 ~
> 
> 추가적으로, document 객체에 직접 접근하는 것을 최대한 "지양" 하는 것이 좋다고 생각하는 데, 리뷰하는 시점에서 대안책이 생각이 안나지만, 방법이 있다면 대체해보는 것도 좋을 것 같아요 ~

사실 이거 방법 찾아보느라 리뷰 반영이 좀 늦었는데, 현재까지는 Ref 를 사용하는 방안이 최선,, 근데 일단 Footer 컴포넌트에 Ref 적용해서 Ref 적용된 컴포넌트로 이동한다 라는 로직.. 을 사용하려면 입점신청 페이지 컴포넌트에서 함수를 만들어서 탑바텀으로 넘겨주는 식으로 해야할 것 같아서 다른 방법 있는지 찾아보고 추후 수정해보겠습니다.. 일단 추천해준 대로 함수 바꿔놓으께 ~